src/scripts/main.js

```diff
@@ -1,3 +1,332 @@
 'use strict';
 
-// write code here
+const tableHeaders = document.querySelectorAll('th');
```

Comment:
It's a good practice to avoid using `document.querySelectorAll` unnecessarily. If these elements are not going to change during the execution of the script, consider using `document.getElementsByTagName` which is faster and returns a live HTMLCollection.